etna/libs/tstcc/augmentations.py

```diff
@@ -66,7 +66,7 @@ def permutation(x, max_segments=5, seg_mode="random"):
             else:
                 splits = np.array_split(orig_steps, num_segs[i])
             # add `np.asarray(splits, dtype=object)` instead of `splits` due to warning about different length of arrays
-            warp = np.concatenate(np.random.permutation(np.asarray(splits, dtype=object))).ravel()
+            warp = np.concatenate(np.random.permutation(np.asarray(splits, dtype=object))).ravel().astype(float)
```

Comment:
Why do we need this?

Reply:
<img width="864" alt="image" src="https://github.com/etna-team/etna/assets/82458209/077de437-e5da-4a9a-b6ce-fe7cc0ccaa1a">

The error was that, I can't remember what it depends on and why it hasn't raised before